src/pages/api/journeyInstances/createNew.ts

```diff
@@ -0,0 +1,68 @@
+import { NextApiRequest, NextApiResponse } from 'next';
+
+import { createApiFetch } from 'utils/apiFetch';
+import { ZetkinPerson } from 'types/zetkin';
+
+const createNewInstance = async (
+  req: NextApiRequest,
+  res: NextApiResponse
+): Promise<void> => {
+  const {
+    query: { orgId, journeyId },
+    method,
+    body,
+  } = req;
+
+  // Return error if method other than POST
+  if (method !== 'POST') {
+    res.setHeader('Allow', ['POST']);
+    res.status(405).end(`Method ${method} Not Allowed`);
+    return;
+  }
+
+  const apiFetch = createApiFetch(req.headers);
+
+  try {
+    const instRes = await apiFetch(
+      `/orgs/${orgId}/journeys/${journeyId}/instances`,
+      {
+        body: JSON.stringify({
+          opening_note: body.note,
+          title: body.title,
+        }),
+        headers: {
+          'Content-Type': 'application/json',
+        },
+        method: 'POST',
+      }
+    );
+
+    const instData = await instRes.json();
+    const instId = instData.data.id;
+
+    const putPeople = async (field: string, people: ZetkinPerson[]) => {
```

Comment:
the field argument could be typed as a union of allowed values. just safer!

```
const putPeople = async (field: 'assignees' | 'subjects', people: ZetkinPerson[]) => {
```

Reply:
Good idea!